.github/workflows/pr-to-update-winget.yml

```diff
@@ -0,0 +1,129 @@
+name: PR for updating winget
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  winget-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout winget-automation
+        uses: actions/checkout@v4
+        with:
+          ref: winget-automation
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          user=${{github.actor}}
+          if [ -z $user ]; then
+            user=vyncent-t
+          fi
+          git config --global user.name "$user"
+          git config --global user.email "$user@users.noreply.github.com"
+
+      - name: Create node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+
+      - name: Set latest tag
+        run: |
+          echo "Setting latest tag"
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          # Remove the 'v' from the tag
+          latestTag=${latestTag#v}
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+
+      - name: Install winget dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/app/windows/winget
+          npm install
+
+      - name: Run winget script
+        run: |
+          echo "Running winget script"
+          echo "Repository: ${{ github.repository }}"
+          echo "Workspace: ${GITHUB_WORKSPACE}"
+          echo $GITHUB_WORKSPACE
+          pwd
+          echo "LATEST_HEADLAMP_TAG: $LATEST_HEADLAMP_TAG"
+          echo "LATEST_HEADLAMP_TAG: ${LATEST_HEADLAMP_TAG}"
+          echo ${LATEST_HEADLAMP_TAG}
+          cd $GITHUB_WORKSPACE/app/windows/winget
+          node winget-create.js $LATEST_HEADLAMP_TAG $GITHUB_WORKSPACE/app/windows/winget
```

Comment:
I think you can call this command adding the desination where we need the files directly (the winget-pkgs).